features/src/main/feature/camel-features.xml

```diff
@@ -1414,9 +1414,9 @@
     <feature name="camel-jq" version="${project.version}" start-level="50">
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <feature version='[2.16,2.17)'>jackson</feature>
-        <bundle dependency='true'>wrap:mvn:net.thisptr/jackson-jq/${jackson-jq-version}$overwrite=merge&amp;Export-Package=net*;version=${jackson-jq-version}</bundle>
-        <bundle dependency='true'>wrap:mvn:org.jruby.jcodings/jcodings/1.0.58</bundle>
-        <bundle dependency='true'>wrap:mvn:org.jruby.joni/joni/2.2.1</bundle>
+        <bundle dependency='true'>wrap:mvn:net.thisptr/jackson-jq/${jackson-jq-version}$overwrite=merge&amp;Bundle-Version=1.0.0.preview_20240207&amp;Export-Package=net*;version=${jackson-jq-version}</bundle>
```

Comment:
same here, if you didn't use the maven property because of the format of the version, so let's not setting it

Reply:
The version coming from the version placeholder doesn't work
`<jackson-jq-version>1.0.0-preview.20240207</jackson-jq-version>`
```
karaf@root()> feature:install camel-jq
org.osgi.framework.BundleException: Could not create bundle object.
        at org.apache.felix.framework.Felix.installBundle(Felix.java:3339)
        at org.apache.felix.framework.BundleContextImpl.installBundle(BundleContextImpl.java:147)
        at org.eclipse.equinox.internal.region.BundleIdBasedRegion.installBundle0(BundleIdBasedRegion.java:120)
        at org.eclipse.equinox.internal.region.BundleIdBasedRegion.installBundleAtLocation(BundleIdBasedRegion.java:100)
        at org.apache.karaf.features.internal.service.BundleInstallSupportImpl.installBundle(BundleInstallSupportImpl.java:135)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.installBundle(FeaturesServiceImpl.java:1145)
        at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:906)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1069)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:1004)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: invalid version "1.0.0-preview.20240207": non-numeric "0-preview"
        at org.osgi.framework.Version.parseInt(Version.java:169)
        at org.osgi.framework.Version.<init>(Version.java:134)
        at org.osgi.framework.Version.valueOf(Version.java:255)
        at org.osgi.framework.Version.parseVersion(Version.java:226)
        at org.apache.felix.framework.util.manifestparser.ManifestParser.<init>(ManifestParser.java:129)
        at org.apache.felix.framework.BundleRevisionImpl.<init>(BundleRevisionImpl.java:117)
        at org.apache.felix.framework.BundleImpl.createRevision(BundleImpl.java:1290)
        at org.apache.felix.framework.BundleImpl.<init>(BundleImpl.java:114)
        at org.apache.felix.framework.Felix.installBundle(Felix.java:3287)
        ... 12 more
Caused by: java.lang.NumberFormatException: For input string: "0-preview"
        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
        at java.base/java.lang.Integer.parseInt(Integer.java:668)
        at java.base/java.lang.Integer.parseInt(Integer.java:786)
        at org.osgi.framework.Version.parseInt(Version.java:167)
        ... 20 more
```